index.html

```diff
@@ -29,6 +30,7 @@ <h2 class="subtitle">Which type of message?</h2>
         <button class="personal-button">Customize</button>
         <button class="clear-message hidden">Clear Message</button>
       </form>
+      <!-- MESSAGE FORM -->
```

Comment:
If you're going to create section comments like this, it would make sense to do it throughout the document.